tests/e2e/specs/infobox.spec.js

```diff
@@ -24,7 +25,7 @@ describe('The infobox', () => {
                 beforeEach(() => {
                     const layer = 'test.wmts.layer'
                     cy.intercept('**/MapServer/identify**', features)
-                    cy.intercept('**/MapServer/**/htmlPopup**', htmlPopup)
+                    cy.intercept('**/MapServer/**/htmlPopup**', htmlPopupTable)
```

Comment:
Oops I didn't see that before, to use fixtures with Cypress you don't need to import them
```
    cy.intercept('**/MapServer/**/htmlPopup**', {
        fixture: 'html-popup.fixture',
    })
```
but then you need to modify the fixture file so it is not a JS file but a plain text file (or more a HTML file in your case)

Reply:
It would then be better to name it `html-popup.html` though...

Reply:
True. I forgot about the intercept property. Thank you! I'll update it.

So. Not `html-popup.fixture.html`?

Reply:
I don't know if it makes sense, it's already located in the `fixtures` folder anyway. See if the other files there have a `.fixture.` part, if not there's no need 😉

Reply:
:+1: